### PR TITLE
BW-1390 Porting postRun endpoint

### PR DESCRIPTION
**Pull request by kpierre13**
…point

### CODE REVIEW

File: common/openapi.yml [hunk 1]
```diff
@@ -123,6 +123,14 @@ components:
         workflow_params:
           type: object
           description: Input set
+#        workflow_attachments:
```
Comment (mspector) [inline common/openapi.yml:126]: are these commented lines included here on purpose? (perhaps as a reminder of some kind?)

**Review by mspector — APPROVED** [review]
Now I see the commit message that says the comments were intentional!

LGTM. If you wanted, you could add a one-liner comment that explains why you're leaving those commented configs in there. But you can judge the importance of that better than me.

**Review by cjllanwarne — COMMENTED** [review]
Current changes LGTM, but I think we also need to update the `RunSetsApiController`

File: service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java [hunk 2]
```diff
@@ -90,31 +92,16 @@ public ResponseEntity<RunStateResponse> postRun(String workflowUrl, Object workf
...
-          null,
-          null,
-          null,
-          null,
-          null,
-          null,
-          null,
-          null,
-          null);
+      wesApi.runWorkflow(workflowParams.toString(), null, null, null, null, workflowUrl, null);
```
Comment (salonishah11) [inline service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:100]: Would you need to update the Cromwell client with changes from [PR 6837](https://github.com/broadinstitute/cromwell/pull/6837) before merging this PR?

Comment (kpierre13) [reply service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java:100]: I don't think so. The problem was the way that Wes was submitting the workflows on the cromwell end, so nothing on the client side would need to be updated that I'm aware of.

**Review by cjllanwarne — APPROVED** [review]

**Review by salonishah11 — APPROVED** [review]

**Pull request merged**